po/fr.po

```diff
@@ -26909,847 +26871,947 @@ msgid "set the list of administrators for GROUP"
 "permissions"
 msgstr ""
+"Supprimer l’avertissement à propos de permissions non sûres sur le fichier "
+"et le dossier de l’utilisateur"
 
 #: /tmp/fish/implicit/share/completions/gpg.fish:169
 msgid "Suppress the warning about missing MDC integrity protection"
 msgstr ""
+"Supprimer l’avertissement à propos de l’absence de système de protection de "
+"l’intégrité"
 
 #: /tmp/fish/implicit/share/completions/gpg.fish:170
 msgid "Refuse to run if GnuPG cannot get secure memory"
-msgstr ""
+msgstr "Refuser l’exécution si GnuPG ne peut obtenir de mémoire sécurisée"
 
 #: /tmp/fish/implicit/share/completions/gpg.fish:171
 msgid "Do not refuse to run if GnuPG cannot get secure memory (default)"
 msgstr ""
+"Ne pas refuser l’exécution si GnuPG ne peut obtenir de mémoire sécurisée "
+"(par défaut)"
 
 #: /tmp/fish/implicit/share/completions/gpg.fish:172
 msgid "Assume the input data is not in ASCII armored format"
-msgstr ""
+msgstr "Présumer que les données d’entrée ne sont pas en armure ASCII"
 
 #: /tmp/fish/implicit/share/completions/gpg.fish:173
 msgid "Do not add the default keyrings to the list of keyrings"
-msgstr ""
+msgstr "Ne pas ajouter les trousseaux par défaut à la liste des trousseaux"
 
 #: /tmp/fish/implicit/share/completions/gpg.fish:174
 msgid "Skip the signature verification step"
-msgstr ""
+msgstr "Passer la vérification de la signature"
 
 #: /tmp/fish/implicit/share/completions/gpg.fish:175
 msgid "Print key listings delimited by colons"
-msgstr ""
+msgstr "Afficher les listes de clés en les délimitant par des doubles points"
 
 #: /tmp/fish/implicit/share/completions/gpg.fish:176
 msgid ""
 "Print key listings delimited by colons (like --with-colons) and print the "
 "public key data"
 msgstr ""
+"Afficher les listes de clés en les délimitant par des doubles points (comme "
+"avec --with-colons) et afficher les données de clé publique"
 
 #: /tmp/fish/implicit/share/completions/gpg.fish:177
+#, fuzzy
 msgid ""
 "Same as the command --fingerprint but changes only the format of the output "
 "and may be used together with another command"
 msgstr ""
+"Pareil que la commande --fingerprint mais ne changer que le format de "
+"sortie ; peut être utilisé avec une autre commande"
 
 #: /tmp/fish/implicit/share/completions/gpg.fish:178
 msgid "Changes the output of the list commands to work faster"
-msgstr ""
+msgstr "Change la sortie des commandes de listage pour travailler plus vite"
 
 #: /tmp/fish/implicit/share/completions/gpg.fish:179
 msgid ""
 "Do not merge primary user ID and primary key in --with-colon listing mode "
 "and print all timestamps as UNIX timestamps"
 msgstr ""
+"Ne pas fusionner l’identifiant utilisateur primaire et la clé primaire dans "
+"les listes avec --with-colon, et afficher tous les horodatages au format UNIX"
 
 #: /tmp/fish/implicit/share/completions/gpg.fish:180
 msgid ""
 "Changes the behaviour of some commands. This is like --dry-run but different"
```

Comment:
W.T.F.‽ (OT, ignore this)

Reply:
This is how I better translated this sentence; I didn't entirely understand it, though, and the `gpg` man was not very useful, so it could probably be improved.

Reply:
No, your translation is just fine, my remark was aimed at “this is like --dry-run, but different”, a sentence I find _éberluante_ – hence me marking it as OT (off topic). I should have deleted that comment; apologies for the confusion.

Reply:
OK, then.